aspnetcore/grpc/performance.md

```diff
@@ -156,6 +156,32 @@ The preceding code configures a channel that sends a keep alive ping to the serv
 
 :::moniker-end
 
+## Flow control
+
+HTTP/2 flow control is a feature that prevents apps from being overwhelmed with data. When using flow control:
+
+* Each app has an available buffer window. The buffer window is how much data the app can receive at once.
```

Comment:
Each connection has a buffer window, and each stream also has one. The app does not.

Reply:
I'm guessing "app" here means invocation of the middleware/procedure, but I think saying that the limit is for both the connection and request is clearer and more precise.